services/ui-src/src/types/index.ts

```diff
@@ -131,6 +131,37 @@ export interface TimeShape {
   second: number;
 }
 
+// Widgets
+export interface iconlessWidget {
```

Comment:
Type interfaces are usually capitalized to keep them separate from regular variables.